statsig_rum/README.md

```diff
@@ -0,0 +1,26 @@
+# Agent Check: statsig-rum
+
+## Overview
+
+Statsig's feature flag tracking integration enriches your Datadog RUM data with feature gate information, allowing you to measure causality between your product features and your system and performance metrics.
```

Comment:
i completed this

Reply:
We try to avoid marketing material/copy. Could you include the above so it's clear for a potential new user to understand statisg and what the integration does?

Reply:
Wanted to bump this!

Reply:
done